packages/yourloops/app/app.tsx

```diff
@@ -45,6 +45,7 @@ const Yourloops = (): JSX.Element => {
       clientId={appConfig.AUTH0_CLIENT_ID}
       redirectUri={window.location.origin}
       useRefreshTokens
+      audience="http://api-public:3000"
```

Comment:
After discussions, we decided to set the same audience for all auth0 tenants, since it won't be checked in the backend but required to have a valid token.